src/main/java/com/github/pedrovgs/androidwifiadb/view/AndroidDevicesTableModel.java

```diff
@@ -0,0 +1,126 @@
+package com.github.pedrovgs.androidwifiadb.view;
+
+import com.github.pedrovgs.androidwifiadb.model.Device;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.swing.table.AbstractTableModel;
+
+public class AndroidDevicesTableModel extends AbstractTableModel {
+    private static final int COLUMN_DEVICE = 0;
+    private static final int COLUMN_STATE = 1;
+    private static final int COLUMN_ACTION = 2;
+
+    private List<Device> devices;
+
+    public AndroidDevicesTableModel() {
+        devices = new ArrayList<Device>();
+    }
+
+    @Override
+    public String getColumnName(int column) {
+        String value;
+        switch (column) {
+            case COLUMN_DEVICE:
+                value = "Device";
+                break;
+            case COLUMN_STATE:
+                value = "State";
+                break;
+            case COLUMN_ACTION:
+                value = "Action";
+                break;
+            default:
+                return null;
+        }
+        return value;
+    }
+
+    @Override
+    public Class<?> getColumnClass(int columnIndex) {
+        Class value = Object.class;
+        switch (columnIndex) {
+            case COLUMN_DEVICE:
+                value = String.class;
+                break;
+            case COLUMN_STATE:
+                value = String.class;
+                break;
+            default:
+                return value;
+        }
+        return value;
+    }
+
+    @Override
+    public int getRowCount() {
+        return devices.size();
+    }
+
+    @Override
+    public int getColumnCount() {
+        return 3;
+    }
+
+    @Override
+    public Object getValueAt(int rowIndex, int columnIndex) {
+        Device obj = devices.get(rowIndex);
+        Object value = null;
+        switch (columnIndex) {
+            case COLUMN_DEVICE:
+                value = obj.toString();
+                break;
+            case COLUMN_STATE:
+                value = obj.isConnected() ? "connected" : "disconnected";
+                break;
+            case COLUMN_ACTION:
+                break;
+            default:
+                return null;
```

Comment:
Can you replace this ``default`` case with a log trace? The ``value`` variable is initialized with null and this variable will be returned if the ``default`` case is executed.